src/main/java/org/example/daos/JobRoleDao.java

```diff
@@ -12,30 +13,55 @@
 import java.util.List;
 
 public class JobRoleDao {
-    public List<JobRole> getJobRoles() throws SQLException {
-        List<JobRole> jobRoles = new ArrayList<>();
+    private static final int STATUS_ID = 1;
+    private static final int ONE = 1;
+    private static final int TWO = 2;
+    private static final int THREE = 3;
+    public List<JobRoleResponse> getOpenJobRoles(
+            final int offset, final int limit)
+            throws SQLException {
+        List<JobRoleResponse> jobRoles = new ArrayList<>();
         try (Connection connection = DatabaseConnector.getConnection()) {
             String query = "SELECT jobRoleId, roleName, location,"
                     + " Capability.capabilityName, Band.bandName, "
                     + "closingDate FROM `job-roles`"
                     + " JOIN Capability using(capabilityId)"
                     + " JOIN Band using(bandId)"
-                    + " WHERE statusId = 1;";
+                    + " WHERE statusId = ?"
+                    + " ORDER BY jobRoleId"
+                    + " LIMIT ? OFFSET ?;";
             PreparedStatement statement = connection.prepareStatement(query);
+            statement.setInt(ONE, STATUS_ID);
+            statement.setInt(TWO, limit);
```

Comment:
offset these indexes... this will now be 1, and the next one 2

Reply:
Resolved